functests/2_performance_update/updating_profile.go

```diff
@@ -159,6 +159,86 @@ var _ = Describe("[rfe_id:28761][performance] Updating parameters in performance
 		})
 	})
 
+	Context("Verify hugepages count split on two NUMA nodes", func() {
+		hpSize2M := performancev2.HugePageSize("2M")
+		t := true
```

Comment:
nit: you can use pointer library to create pointers without the need to define new variables, see for example https://github.com/openshift-kni/performance-addon-operators/blob/50d3c1067cff0d74267e7a356d163ba09187e702/functests/1_performance/cpu_management.go#L428